app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+class CopyFileError(Exception):
+    pass
+
+
+class InvalidCommandError(CopyFileError):
+    pass
+
+
+class SameFileError(CopyFileError):
+    pass
+
+
+class CustomFileNotFoundError(CopyFileError):
+    pass
+
+
+class FileCopyError(CopyFileError):
+    pass
+
+
+def copy_file(copy_command: str) -> None:
+    command, source_file, destination_file = copy_command.split()
```

Comment:
Would make sense to check that len of command is equal to 3 first. Otherwise your program will crash during unpacking